### PR TITLE
chore(carbonserver) rework api endpoints

### PR DESCRIPTION
**Pull request by prmths128**
Rework the remaining api endpoints to better represent the concepts

=> https://github.com/orgs/mlco2/projects/4/?pane=issue&itemId=67359449

### CODE REVIEW

File: carbonserver/carbonserver/api/routers/emissions.py [hunk 1]
```diff
@@ -77,3 +79,15 @@ def get_emissions_from_run(
...
+@router.get("/emissions", tags=EMISSIONS_ROUTER_TAGS, response_model=Page[Emission])
+@inject
+def get_emissions(
+    run_id: str,
+    emission_service: EmissionService = Depends(
+        Provide[ServerContainer.emission_service]
+    ),
+    params: Params = Depends(),
+) -> Page[Emission]:
+    return paginate(emission_service.get_emissions_from_run(run_id), params)
```
Comment (inimaz) [inline carbonserver/carbonserver/api/routers/emissions.py:93]: what if no run_id is provided in the query? How to handle that?

**Review by inimaz — COMMENTED** [review]
Minor comment, but other than that looks good, thanks!

File: carbonserver/carbonserver/api/routers/emissions.py (hunk 1 above)
Comment (prmths128) [reply carbonserver/carbonserver/api/routers/emissions.py:93]: run_id is mandatory so you can access emissions by either:
- /runs/{runs[0]['id']}/emissions
- /emissions?run_id=...

If you don't specify run_id in the second variant, you'll get a fastapi validation error

**Review by inimaz — APPROVED** [review]
Looks good, thanks!

**Pull request merged**